src/layer/marker/Marker.Drag.js

```diff
@@ -58,10 +58,11 @@ L.Handler.MarkerDrag = L.Handler.extend({
 		}
 
 		marker._latlng = latlng;
+		e.latlng = latlng;
 
 		marker
-		    .fire('move', {latlng: latlng})
-		    .fire('drag');
+		    .fire('move', e)
```

Comment:
Also, as we are doing two forwards, I'm not sure at which step we should introduce the `originalEvent` key ;)